raphtory/src/db/graph/views/filter/node_name_filtered_graph.rs

```diff
@@ -31,14 +37,14 @@ impl<G> NodeNameFilteredGraph<G> {
     }
 }
 
-impl CreateFilter for NodeNameFilter {
-    type EntityFiltered<'graph, G: GraphViewOps<'graph>> = NodeNameFilteredGraph<G>;
+impl<G: GraphView> NodeOp for NodeNameFilteredGraph<G> {
+    type Output = bool;
 
-    fn create_filter<'graph, G: GraphViewOps<'graph>>(
-        self,
-        graph: G,
-    ) -> Result<Self::EntityFiltered<'graph, G>, GraphError> {
-        Ok(NodeNameFilteredGraph::new(graph, self.0))
+    fn apply(&self, storage: &GraphStorage, node: VID) -> Self::Output {
```

Comment:
you should turn this into a NodeOP (it doesn't need to know about the graph and this can be simplified)